README.md

```diff
@@ -53,20 +63,24 @@ If you want to use this with just the single configuration file, make sure to in
 
 ## `token`
 
-[Generate a personal access token](https://github.com/settings/tokens), with the `repo:public_repo` scope for only public repositories or the `repo` scope for public and private repositories, and add it to _Secrets_ (repository settings) as `RENOVATE_TOKEN`. You can also create a token without a specific scope, which gives read-only access to public repositories, for testing. This token is only used by Renovate, see the [token configuration](https://docs.renovatebot.com/self-hosted-configuration/#token), and gives it access to the repositories. The name of the secret can be anything as long as it matches the argument given to the `token` option.
+[Generate a Personal Access Token](https://github.com/settings/tokens), with the `repo:public_repo` scope for only public repositories or the `repo` scope for public and private repositories, and add it to _Secrets_ (repository settings) as `RENOVATE_TOKEN`.
+You can also create a token without a specific scope, which gives read-only access to public repositories, for testing.
+This token is only used by Renovate, see the [token configuration](https://docs.renovatebot.com/self-hosted-configuration/#token), and gives it access to the repositories.
+The name of the secret can be anything as long as it matches the argument given to the `token` option.
 
-Note that the [`GITHUB_TOKEN`](https://help.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token#permissions-for-the-github_token) secret can't be used for authenticating Renovate because it has too restrictive permissions. In particular, using the `GITHUB_TOKEN` to create a new `Pull Request` from more types of Github Workflows results in `Pull Requests` that [do not trigger your `Pull Request` and `Push` CI events](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#using-the-github_token-in-a-workflow).
+Note that the [`GITHUB_TOKEN`](https://help.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token#permissions-for-the-github_token) secret can't be used for authenticating Renovate because it has too restrictive permissions.
+In particular, using the `GITHUB_TOKEN` to create a new `Pull Request` from more types of Github Workflows results in `Pull Requests` that [do not trigger your `Pull Request` and `Push` CI events](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#using-the-github_token-in-a-workflow).
 
 If you want to use the `github-actions` manager, you must setup a [special token](#special-token-requirements-when-using-the-github-actions-manager) with some requirements.
 
 ## `renovate-version`
```

Comment:
Copy/pasting a comment from a user: [^quote]

> With this in mind, do you recommend that users simply pick `uses: renovatebot/github-action@v36.0.0` and not pin `renovate-version`?
> 
> I am guessing the idea is to get the latest non-breaking 36.x version. And if you make a new major, we simply upgrade `uses: renovatebot/github-action@v37.0.0`. Is that it?
> 
> > use [renovate-version](https://github.com/renovatebot/github-action#renovate-version) to pin the renovate version.
> 
> The only reason I am in doubt is this ☝️ I would only use that if you accidentally released something that didn't work for our project, does that sound about right? 🤔

What do we recommend users do with `renovate-version`? I would think pinning is best, as then you get a "Update `renovate-version` PR" from GitHub Actions Renovate? You're not "surprised" by "sudden" breaking changes in the upstream Renovate program that way.

[^quote]: https://github.com/renovatebot/renovate/discussions/21577#discussioncomment-5683354

Reply:
we probably should suggest pin and provide a regex manager to auto update as preset